src/app/app-routing.module.ts

```diff
@@ -30,6 +30,10 @@ const routes: Routes = [
     path: 'attributes',
     loadChildren: () => import('./pages/attributes-page/attributes.page.module').then( m => m.AttributesPageModule)
     ,canActivate: [AuthGuard]
+  },
+  {
+    path: 'permissions-page',
```

Comment:
Always look at the other code around your addition, and be sure it is in the same style. This path should be **permissions**, remove the -page